integration-tests/__tests__/coverage_transform_instrumented.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const {readFileSync} = require('fs');
+const path = require('path');
+const skipOnWindows = require('../../scripts/SkipOnWindows');
+const {cleanup, run} = require('../utils');
+const runJest = require('../runJest');
+
+const dir = path.resolve(__dirname, '../coverage-transform-instrumented');
+const coverageDir = path.join(dir, 'coverage');
+
+skipOnWindows.suite();
+
+beforeAll(() => {
+  cleanup(coverageDir);
+});
+
+it('code coverage for transform instrumented code', () => {
```

Comment:
I'm not sure if this your intention or not, but on a hunch I pulled this test out and ran it against master and it passed. So if you're trying to test some change in this PR, something should change. I'm still not sure what exactly you're going for here. If you're just trying to prevent future regressions for this use case, I suppose it's fine!

Reply:
There was no test checking for coverage when used with a transform, that's why I added it.

Is there some other scenario that I could write test for too?

When I started, I thought coverage remapping was broken in master, but it turns out that it is not.